Dalamud/IoC/Internal/ServiceContainer.cs

```diff
@@ -127,7 +128,26 @@ public void RegisterInterfaces(Type type)
             return null;
         }
 
-        ctor.Invoke(instance, resolvedParams);
+        var tcs = new TaskCompletionSource(TaskCreationOptions.RunContinuationsAsynchronously);
```

Comment:
Could you leave a comment here explaining why this is necessary?

Reply:
I'm not sure if this fixes problems; more of seeing if this fixes stalling problems on load time.

Reply:
Alright, let's leave it like this for now and see if things work out